packages/mds-compliance/mds-compliance-engine.ts

```diff
@@ -281,39 +273,36 @@ function processPolicy(
             return [...acc, ...map.matched]
           }, [])
 
-          const overflowVehicles = bucketMap.reduce((acc: (MatchedVehicle & { rule_id: UUID })[], map) => {
-            return [...acc, ...map.overflowed]
-          }, [])
+          bucketMap.forEach(bucket => {
```

Comment:
This is the textbook definition of what a `reduce` function should be doing, I don't like this change.